IntelliTect.TestTools.TestFramework/IntelliTect.TestTools.TestFramework/TestBuilder.cs

```diff
@@ -177,15 +175,24 @@ public void ExecuteTestCase()
                     }
                     TestBlockException = tempException;
                 }
-            }
 
+                if(TestBlockException == null)
+                {
+                    logger?.Info("Test case finished successfully.");
+                }
+                else
+                {
+                    logger?.Error($"Test case failed: {TestBlockException.Message}");
+                    logger?.Error($"Stacktrace: {TestBlockException.StackTrace}");
```

Comment:
I would recommend just using `TestBlockException.ToString()`. Though what you have covers most exceptions reasonably some do add additional properties with information.

Reply:
Done and done.